docs/conf.py

```diff
@@ -73,7 +73,7 @@
 
 # General information about the project.
 project = u'Watchmaker'
-copyright = u'2016, Plus3 IT Systems'  # noqa: A001
```

Comment:
Copyright matches the LICENSE. I haven't seen anything convincing to indicate it's really necessary to update the year.....

Reply:
Four points: 1. legally, everything you reduce to tangible medium is protected by copyright regardless, even if you have no copyright notice. 2. having an up-to-date copyright year does provide an educational/legal advantage in that it shows you actively protect you intellectual property. people in cyberspace may erroneously think you've abandoned your work and they can help themselves (they are legally and technically wrong). 3. having the date up-to-date shows that the project is fresh, still being maintained, that the maintainers care, and 4. if you want to show the longstanding nature of a project, it is perfectly acceptable to put a range (e.g., (c) Plus3 IT Systems 2009-2018).

Reply:
Understood, we can consider it separately. I'd have a concern about maintaining the year range across all our projects... Absent a solution for that, I'd be more inclined to just removing the year.